### PR TITLE
Fixes part of #180: Fix issue bot not commenting on pull request from new contributors

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation

Fixes part of #180: Adds a try-catch block when checking if the user is a collaborator. Probot throws an error when the Github API responds with a 404, and that causes the bot to not comment on the pull request appropriately. This PR adds a try-catch block to prevent such scenarios.

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -219,6 +219,27 @@ const isUserAMemberOfTheOrganisation = async (context, username) => {
   }
 };
 
+/**
+ * Check if the user is a collaborator.
+ *
+ * @param {import('probot').Context} context
+ * @param {string} username
+ */
+const isUserACollaborator = async (context, username) => {
```
Comment (vojtechjelinek) [inline lib/utils.js:228]:
```suggestion
const isUserCollaborator = async (context, username) => {
```

File: lib/utils.js [hunk 2]
```diff
@@ -280,9 +301,9 @@ const isChangelogLabel = (label) => {
  * @param {string} type
  */
 const getLastReviewOfSpecificType = async (
-  context,
-  type,
-  pullRequestNumber
+    context,
+    type,
+    pullRequestNumber
```
Comment (vojtechjelinek) [inline lib/utils.js:306]: Why this change?

File: lib/utils.js [hunk 3]
```diff
@@ -177,8 +177,8 @@ const hasPullRequestBeenApproved = async (context, pullRequestNumber) => {
  * @param {Number} pullRequestNumber
  */
 const doesPullRequestHaveChangesRequested = async (
-  context,
-  pullRequestNumber
+    context,
+    pullRequestNumber
```
Comment (vojtechjelinek) [inline lib/utils.js:181]: Why this change?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Few nits. Did you change if there are any other places where errors like this can occur?

File: lib/utils.js (hunk 3 above)
Comment (jameesjohn) [reply lib/utils.js:181]: I think it's due to the eslint errors I was getting locally.

File: lib/utils.js (hunk 2 above)
Comment (jameesjohn) [reply lib/utils.js:306]: It's due to the eslint errors I was getting locally.

**Comment by jameesjohn** [comment]
> Did you change if there are any other places where errors like this can occur?

Yes. This is the only place left.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Comment by vojtechjelinek** [comment]
@jameesjohn Please release this.
